src/Npgsql/TypeHandlers/ByteaHandler.cs

```diff
@@ -12,8 +12,15 @@ namespace Npgsql.TypeHandlers
     /// <remarks>
     /// http://www.postgresql.org/docs/current/static/datatype-binary.html
     /// </remarks>
-    [TypeMapping("bytea", NpgsqlDbType.Bytea, DbType.Binary, new[] { typeof(byte[]), typeof(ArraySegment<byte>) })]
+    [TypeMapping(
+        "bytea",
+        NpgsqlDbType.Bytea,
+        DbType.Binary,
+        new[] { typeof(byte[]), typeof(ReadOnlyMemory<byte>), typeof(Memory<byte>), typeof(ArraySegment<byte>) })]
```

Comment:
Should this be put into a conditional compilation block too? Or should conditions be completely removed?

Reply:
Is there a reason for conditional compilation here? Memory and ROM are supported back on netstandard2.0 via the System.Memory nuget (even if they're less efficient).

Reply:
Just don't understand why do we need `#if !NETSTANDARD2_0 && !NET461` block everywhere except this place?

Reply:
AFAIK all places where we have `#if`, it's because something isn't supported on netstandard2.0/net461 - this is simply not the case here, it's supported. Can you point to an `#if` which you don't understand?

Reply:
Sorry, my fault! I already forgot the context of this PR, you're right - we shouldn't register Memory on TFMs where it can't be written...!